lang/csharp/src/apache/test/Avro.test.csproj

```diff
@@ -42,7 +43,7 @@
     </PackageReference>
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)'!='netcoreapp2.2'">
+  <ItemGroup Condition="'$(TargetFramework)'!='netcoreapp3.1'">
```

Comment:
Which version of **Microsoft.NET.Test.Sdk** do we need to reference for **netcoreapp3.1**? We have another condition check for **netcoreapp2.2** a few lines down, on line 50. Before, the logic was

```
if netcoreapp2.2
    use v15.3.0
else
    use v15.6.1
```

I'm surprised that it's actually getting a reference to **Microsoft.NET.Test.Sdk** when targeting **netcoreapp3.1** right now with the current conditions. I must be missing something.

Reply:
It is referencing the 15.6.1 version which is not the latest. The latest Test.Sdk version of 16.4.0 can be used without any netcoreapp version checking